### ISSUE TITLE
LiveReload Chrome Extension

### ISSUE THREAD

**Issue by quangv** [tomekwi/elm-live#113]
Hi there, I dev using VirtualBox, so I use `--host=0.0.0.0` 

I've never been able to just use a Chrome extension to get LiveReload to work.

Is there a way to use a plugin/extension in Chrome to get LiveReload to work?

Thanks.

**Comment by tomekwi** [comment]
Yeah, this isn’t supposed to work with a LiveReload browser extension. elm-live comes with LiveReload built in and delivered as a JS snippet to the browser.

Your issue seems to be that your host IP is different to the IP that you need to access from the browser? Please have a look at #112 and check if it might be the same problem. If it is, let us know in that issue.

We’re already trying to figure out a solution! 😀
